lib/manageiq/api/client/action.rb

```diff
@@ -0,0 +1,17 @@
+module ManageIQ
+  module API
+    module Client
+      class Action
+        attr_accessor :name
+        attr_accessor :method
+        attr_accessor :href
+
+        def initialize(action_hash)
+          @name   = action_hash["name"]
+          @method = action_hash["method"]
+          @href   = action_hash["href"]
```

Comment:
This can be one-lined, if you prefer to: `@name, @method, @ref = action_hash.values_at("name", "method", "ref")`